src/highlight.js

```diff
@@ -71,7 +71,7 @@ const HLJS = function(hljs) {
     // beta configuration options, subject to change, welcome to discuss
     // https://github.com/highlightjs/highlight.js/issues/1086
     __emitter: TokenTreeEmitter,
-    cssSelector: 'pre code',
```

Comment:
(I'd left the trailing comma on the new line in order for future diffs to be cleaner; I don't know whether the current setting of the linter is correct.)

Reply:
We like clean diffs but we don't like extra commas. :)  They used to **break** things, but that's no longer true.  Perhaps something to revisit in the future, but for now we obey the linter.